### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by cafferata**
Following the Composer best practices and simplified the installation step(s).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,23 +42,11 @@ By Composer
 
 The easiest way to install PredictionIO PHP client is to use [Composer](http://getcomposer.org/).
 
-1. Add `predictionio/predictionio` as a dependency in your project's ``composer.json`` file:
+1. `Predictionio` is available on [Packagist](https://packagist.org/packages/league/csv) and can be installed using [Composer](https://getcomposer.org/):
```
Comment (dszeto) [inline README.md]: Do you mean https://packagist.org/ instead for Packagist's link?

The correct letter case for the module name should be `predictionio`. If indicating the project name, please use Apache PredictionIO.

**Comment by cafferata** [comment]
I have processed both feedback points.

**Comment by cafferata** [comment]
Hi @dszeto, i'm running over my (outstanding) pull request and so I came across this one. Are there open points why this pull request can not be merged? Can I help with something? Do you expect changes and/or feedback processing from me?

**Comment by dszeto** [comment]
Sorry for the delay. This looks good to me now. Thanks @cafferata !

**Pull request closed without merging**